### PR TITLE
chore(macros/LearnSidebar): update zh-CN translation of LearnSidebar

### PR DESCRIPTION
**Pull request by jasonren0403**
## Summary

the same as title

part of https://github.com/mdn/translated-content/issues/13881

cc: @yin1999 @awxiaoxian2020 

### Problem

some zh-CN sidebar strings is missing in learn area sidebar

### Solution

add the missing keys and update existing ones to improve consistency

---

### After

---

### CODE REVIEW

**Review by awxiaoxian2020 — APPROVED** [review]

File: kumascript/macros/LearnSidebar.ejs [hunk 1]
```diff
@@ -164,6 +168,7 @@ const l10nStrings = mdn.localStringMap({
     'Server-side_website_programming' : '服务端网页编程',
       'First_steps' : '第一步',
       'Django_web_framework_(Python)' : 'Django 网站框架 (Python)',
+      'Express_Web_Framework_(Node.js_JavaScript)' : 'Express 网站框架 (node.js/JavaScript)',
```
Comment (yin1999) [inline kumascript/macros/LearnSidebar.ejs:171]:
```suggestion
      'Express_Web_Framework_(Node.js_JavaScript)' : 'Express Web 框架（Node.js/JavaScript）',
```

File: kumascript/macros/LearnSidebar.ejs [hunk 2]
```diff
@@ -164,6 +168,7 @@ const l10nStrings = mdn.localStringMap({
     'Server-side_website_programming' : '服务端网页编程',
       'First_steps' : '第一步',
       'Django_web_framework_(Python)' : 'Django 网站框架 (Python)',
```
Comment (yin1999) [inline kumascript/macros/LearnSidebar.ejs:170]: By the way, could we update the translation here?

```suggestion
      'Django_web_framework_(Python)' : 'Django 网站框架（Python）',
```

File: kumascript/macros/LearnSidebar.ejs [hunk 3]
```diff
@@ -136,21 +136,25 @@ const l10nStrings = mdn.localStringMap({
...
       'Web_forms_core' : 'Web 表单核心',
-      'Web_forms_advanced': 'Web表单进阶',
+      'Web_forms_advanced': 'Web 表单进阶',
     'Accessibility_—_Make_the_web_usable_by_everyone' : '无障碍——使每个人都能使用 Web',
       'Accessibility_guides' : '无障碍指南',
       'Accessibility_assessment' : '无障碍测评',
+    'Performance' : '性能——使网站快速响应',
+      'Performance_guides' : '性能指南',
+    'MathML_Writing_mathematics' : 'MathML——使用 MathML 语言撰写数学表达式',
+      'MathML_first_steps': 'MathML 初步',
```
Comment (yin1999) [inline kumascript/macros/LearnSidebar.ejs:157]: How about align with the translation in line 138?

```suggestion
      'MathML_first_steps': 'MathML 第一步',
```

**Comment by jasonren0403** [comment]
cc @mdn/core-dev for a look 👀

**Comment by awxiaoxian2020** [comment]
@yin1999 It seems ping @mdn/core-dev doesn't work with @jasonren0403 's permission. I notice that the "ping" is not bolded.

**Comment by yin1999** [comment]
> @yin1999 It seems ping `@mdn/core-dev` doesn't work with `@jasonren0403`'s permission. I notice that the "ping" is not bolded.

Hey @awxiaoxian2020. `@mention` is only visible to organization members, I believe jasonren0403 correctly notified the dev team.

**Comment by jasonren0403** [comment]
> Hey @awxiaoxian2020. `@mention` is only visible to organization members, I believe jasonren0403 correctly notified the dev team.

I have, but the dev team still haven't replied 🤔

**Comment by jasonren0403** [comment]
try pinging @LeoMcA for a look 👀

**Comment by caugner** [comment]
Sorry for the long wait, and thank you for these improvements! 🙌

**Pull request merged**